File: pkg/codegen/testing/test/testdata/third-party-package-pp/python/third-party-package.py
import pulumi
import third_party as other

other = other.Thing("Other", idea="Support Third Party")
question = other.module.Object("Question", answer=42)
question2 = other.module.sub.Object("Question2", answer=24)
provider = other.Provider("Provider", object_prop={
    "prop1": "foo",
    "prop2": "bar",
    "prop3": "fizz",
})

File: pkg/codegen/testing/test/testdata/third-party-package-pp/nodejs/third-party-package.ts
import * as pulumi from "@pulumi/pulumi";
import * as other from "@third-party/pulumi-other";

const Other = new other.Thing("Other", {idea: "Support Third Party"});
const Question = new other.module.Object("Question", {answer: 42});
const Question2 = new other.module.sub.Object("Question2", {answer: 24});
const Provider = new other.Provider("Provider", {objectProp: {
    prop1: "foo",
    prop2: "bar",
    prop3: "fizz",
}});
